src/test/java/org/skife/jdbi/v2/sqlobject/TestGetGeneratedKeysPostgres.java

```diff
@@ -25,19 +23,23 @@
 
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.junit.Assert.assertThat;
+import static org.junit.Assume.assumeTrue;
 
-public class TestGetGeneratedKeysPostgres
-{
-    private JdbcConnectionPool ds;
-    private DBI                dbi;
+public class TestGetGeneratedKeysPostgres {
```

Comment:
It might seem petty, but can we revert back to the original brace style? This line change doesn't contribute to the fix, and can cause merge conflicts.